tutorials/commands/flux-proxy-command.rst

```diff
@@ -0,0 +1,194 @@
+.. _flux-proxy-command:
+
+=================
+Flux Proxy Basics
+=================
+
+With Flux, it is very common to create a Flux :ref:`subinstances<subinstance>`.  However, it can be confusing how to interact with those subinstances.
+
+This tutorial will introduce the :core:man1:`flux-proxy` command, which can be used to interact with subinstances.
+
+-----------------------
+Starting a Subinstance
+-----------------------
+
+First, let's create a subinstance to work with.  The most common way is through ``flux mini batch``.  We'll launch a subinstance and run a few jobs under that instance, we then call ``flux job status`` to wait on those jobs to finish.
+
+.. code-block:: sh
+
+   #!/bin/sh
+   #filename: subinstance-jobs.sh
+
+   jobid1=`flux mini submit -n1 sleep inf`
+   jobid2=`flux mini submit -n1 sleep inf`
+   jobid3=`flux mini submit -n1 sleep inf`
+   jobid4=`flux mini submit -n1 sleep inf`
+   flux job status ${jobid1} ${jobid2} ${jobid3} ${jobid4}
+
+.. code-block:: console
+
+   > flux mini batch -n4 ./subinstance-jobs.sh
```

Comment:
I totally don't understand what is going on here, lol (I've never used batch before).

Reply:
OK going to stop here and submit this early feedback because I can't follow along (sorry, not familiar with batch!)

Reply:
lemme try to add a bit more details then.  one issue is we are writing these tutorials out of order.  I jumped on `flux proxy` mostly b/c you already wrote the ssh-across-clusters one. Really `flux mini batch` probably should have come after `flux mini submit`.  which I will add to the todo list.

Reply:
shouldn't we encourage the use of `--cc` in this case? Possibly using waitable jobs and  `flux job wait` for more "standard" practice.

Reply:
for some of these small examples I figure its easier to do this than have to explain `--cc` (leaving `--cc` to the #195).  

Do we have a preference on `flux job wait` vs `flux job status` for something this small?

Reply:
Good points. Given that I'm fine with current example, was more of a question anyway.